commands/db.cmd

```diff
@@ -39,6 +39,10 @@ case "${WARDEN_PARAMS[0]}" in
             | "${WARDEN_DIR}/bin/warden" env exec -T db \
             mysql -u"${MYSQL_USER}" -p"${MYSQL_PASSWORD}" --database="${MYSQL_DATABASE}" "${WARDEN_PARAMS[@]:1}" "$@"
         ;;
+    dump)
+            "${WARDEN_DIR}/bin/warden" env exec -T db \
+            mysqldump -u"${MYSQL_USER}" -p"${MYSQL_PASSWORD}" "${MYSQL_DATABASE}" | pv | gzip > "${WARDEN_PARAMS[@]:1}" "$@"
```

Comment:
1. I believe we should stip all the "definer" (and maybe some global things) we're doing in the import command. What do you think?
2. I think using gzip should be optional, maybe with some flag? I think would be good to have support for flow like this: 
```bash
warden db dump > dump.sql 
pv dump.sql | warden db import
```

Reply:
Hi @ihor-sviziev 
Thanks, good suggestion

Reply:
Let's have it output vanilla mysldump results if we can, as this will allow the dump to be inspected to see what was actually inside the db container. The piping here makes gzip and pv become dependencies (gzip is typically present by default on workstations, but pv often must be installed by a user). 

```suggestion
            mysqldump -u"${MYSQL_USER}" -p"${MYSQL_PASSWORD}" "${MYSQL_DATABASE}" "${WARDEN_PARAMS[@]:1}" "$@"
```

Reply:
As written, it causes errors, so this would need to be updated regardless.
```
davidalger@silver-bullet:~/Sites/m2sandbox (develop) $ warden db dump
/opt/warden/commands/db.cmd: line 44: "${WARDEN_PARAMS[@]:1}": ambiguous redirect
0.00 B 0:00:01 [0.00 B/s] [<=>                                                                                                                                                ]
ERROR: Command `gzip "$@" > "${WARDEN_PARAMS[@]:1}"` at /opt/warden/commands/db.cmd:31 failed with exit code 1
```

Verified the suggested change here functions using `warden db dump` (prints to shell) or `warden db dump | pv | gzip > test.sql.gz` (dumps compressed dump to file with status)